### DISCUSSION TITLE
Collapsing windows or dockable to work with multiple monitors

### DISCUSSION THREAD

**Discussion by 4l3m4oO** [friction2d/friction#46] [Ideas]
Dockable windows/sections are very useful to work, when you have 2 or more monitors, (ex.) putting the preview in one and the timeline and other docks on another monitor. this increase a lot the speed of the workflow.

Maybe implement this and one function "manage workspaces".

**Comment by rodlie** [comment]
The current focus has been on one monitor, but will figure out a multi monitor solution, most likely in the next version (0.9.4+).

I'm currently preparing for a new release (0.9.3, due this weekend) and will only do minor changes.

**Reply by 4l3m4oO** [reply]
Thank you for all replys

**Comment by rodlie** [comment]
Created a new issue #50
